utils/publish.mjs

```diff
@@ -56,7 +56,7 @@ export const collectPackageJsons = (packageNames) => {
 };
 
 export const publishAsNeeded = (packageNames, dryRun = false) => {
-    execSync(`git fetch --tags`);
+    execSync(`git fetch --tags -f`);
```

Comment:
Probably worth adding a comment here essentially copying in the PR message so that future folks know why this is important!